### PR TITLE
Anarchist classics and other changes to philosophy_book snippets

### PR DESCRIPTION
**Pull request by park66665**
#### Summary
 ```SUMMARY: NONE```

#### Purpose of change

1. I felt that there aren't enough anarchist classics; Stirner, Kaczynski, and Thoreau are cool, but I want to see old-school-ish anarchism.
2. I ended up wanting to add some other philosophical books that fit well to the cataclysm IMHO.
3. Lack of Schopenhauer was noted.
4. There were some non-double spaces after periods.

#### Describe the solution

1. Add four important anarchist books.
2. Add some cataclysm-ish books.
2-1. Add a book about nihilism or antinatalism et cetera.
2-2. Add a book about transhumanism.
3. Add Schopenhauer.
4. Add double spaces.

#### Describe alternatives you've considered

0. Just adding double spaces, or adding missing descriptions; some snippets have only one sentence!
1. Add more contemporary schools of thought instead of old, old anarchism, exempli gratia, Digimodernism by Alan Kirby (post-postmodernism), Virtual Worlds as Philosophical Tools by Stefano Gualeni (game philosophy), Gender Trouble by Judith Butler (queer theory...?) et cetera; but they are a little bit _too_ contemporary... I guess?
1-1. Mutual Aid instead of Conquest of Bread?
1-2. Bob Black? How about Saul Newman, John Zerzan, or digging up some obscure contemporary anarchist philosophers?
2. Not doing it? Should I conjure up some fictional books?
2-1. Antinatalism is.. too depressing. How about Voluntary Human Extinction?
2-1-1. A Critique of Affirmative Morality by Julio Cabrera since I read parts of that and know that it fits very well with philosophy_book, but that book is from... well... 1996 (too contemporary).
2-1-2. How about a book from Zapffe?
2-1-3. Benatar?
2-2. Why there aren't enough books about Anarcho-Transhumanism(and its synonyms)? It fits really well to this game!
2-2-1. More contemporary Transhumanism books like The Posthuman Condition by Robert Pepperell, Superintelligence by Nick Bostrom, et cetera.
2-2-2. A Cyborg Manifesto by Donna J. Haraway?
2-2-3. The Artilect War by Hugo de Garis?
3. Not adding Schopenhauer. We already have a few books by Nietzsche! (There isn't Thus Spoke Zarathustra btw)
4. No alternatives here.

Note: Cioran was chosen because Cioran is more "established" in the literary world.

#### Additional context

My first language is not English, so texts need some additional perusal by native English speakers.

I am really open to suggestions since I am not very knowledgeable about philosophy; I didn't even read enough Kant or Hegel! All feedbacks are welcome.

### CODE REVIEW

File: data/json/items/book/misc.json [hunk 1]
```diff
@@ -1033,6 +1033,34 @@
...
+        "id": "philosophy34",
+        "text": "This is a copy of \"Enquiry Concerning Political Justice\" by William Godwin.  This thick book is filled with antiquated phrases."
+      },
+      {
+        "id": "philosophy35",
+        "text": "This is a copy of \"The Abolition of Work and Other Essays\" by Bob Black.  It is likely that \"The Abolition of Work\" is the most famous essay in this book."
+      },
+      {
+        "id": "philosophy36",
+        "text": "This is a copy of \"What is Property?\" by Pierre-Joseph Proudhon.  It looks like that this book has a surprisingly long track record of owners."
```
Comment (anothersimulacrum) [inline data/json/items/book/misc.json]:
```suggestion
        "text": "This is a copy of \"What is Property?\" by Pierre-Joseph Proudhon.  It looks like this book has a surprisingly long track record of owners."
```

Comment (park66665) [reply data/json/items/book/misc.json]: That 'that' indeed looks unnecessary. Thank you for the review!

**Comment by LaVeyanFiend** [comment]
Where's my _No Treason_ by Lysander Spooner?

**Comment by faefux** [comment]
Thanks for doing this, it was nothing intentionally, I literally just added what was on my bookshelf or what was referenced in other books I owned and i'm just not that into commie anarchy.   I highly encourage people to add more books but be considerate that all this has to be translated by somebody,  the original list of snippets for example in my install is much longer. :)

**Pull request merged**